src/main/cpp/fielddata/FieldElementCalculator.h

```diff
@@ -76,90 +89,90 @@ class FieldElementCalculator
     frc::Transform3d m_calcCoralLeftAlliance = frc::Transform3d(
         frc::Translation3d(
             units::length::inch_t(0.0),
```

Comment:
Should we create a m_noXOffset and a m_noZOffset and replace the units::length::inch_t(0.0) here?